doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -295,7 +295,283 @@ Your output should look like the following:
+          }
+        }
+      ]
+    }
+
+Run the following command to create a role with the trust
+relationship defined in \ ``trust.json``. This role enables the
+Amazon EKS cluster to get and refresh credentials from IAM.
+
+::
+
+    aws iam create-role --role-name <role name> --assume-role-policy-document file://trust.json --output=text
+
+Your output should look like the following:
+
+::
+
+    ROLE    arn:aws:iam::123456789012:role/my-role 2019-11-22T21:46:10Z    /       ABCDEFSFODNN7EXAMPLE   my-role
+    ASSUMEROLEPOLICYDOCUMENT        2012-10-17
+    STATEMENT       sts:AssumeRoleWithWebIdentity   Allow
+    STRINGEQUALS    sts.amazonaws.com       system:serviceaccount:my-namespace:sagemaker-k8s-operator-default
+    PRINCIPAL       arn:aws:iam::123456789012:oidc-provider/oidc.eks.us-east-1.amazonaws.com/id/
+
+Take note of \ ``ROLE ARN``, you pass this value to your
+operator. 
+
+Attach the AmazonSageMakerFullAccess Policy to your Role
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To give the role access to Amazon SageMaker, attach
+the \ `AmazonSageMakerFullAccess <https://console.aws.amazon.com/iam/home?#/policies/arn:aws:iam::aws:policy/AmazonSageMakerFullAccess>`__ policy.
+If you want to limit permissions to the operator, you can create your
+own custom policy and attach it.
+
+To attach AmazonSageMakerFullAccess, run the following command:
+
+::
+
+    aws iam attach-role-policy --role-name <role name>  --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+
+The Kubernetes
+ServiceAccount \ ``sagemaker-k8s-operator-default`` should
+have \ ``AmazonSageMakerFullAccess`` permissions. Confirm this when you
+install the operator.
+
+Deploy the Operator to Your Namespace
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When deploying your operator, you can use either a YAML file or Helm
+charts. 
+
+Deploy the Operator to Your Namespace Using YAML
+''''''''''''''''''''''''''''''''''''''''''''''''
+
+There are two parts to deploying an operator within the scope of a namespace. The first is the set of CRDs that are installed at a cluster level. These resource definitions only need to be installed once per Kubernetes cluster. The second part is the operator permissions and deployment itself.
+
+If you have not already installed the CRDs into the cluster, apply the CRD installer YAML using the following command:
+
+::
+
+    kubectl apply -f https://raw.githubusercontent.com/aws/amazon-sagemaker-operator-for-k8s/master/release/rolebased/namespaced/crd.yaml
+    
+To install the operator onto the cluster:
+
+-  Download the operator installer YAML using the following command:
+
+   ::
+
+       wget https://raw.githubusercontent.com/aws/amazon-sagemaker-operator-for-k8s/master/release/rolebased/namespaced/operator.yaml
+
+- Update the installer YAML to place the resources into your specified namespace using the following command:
+
+   ::
+   
+       sed -i -e 's/PLACEHOLDER-NAMESPACE/<YOUR NAMESPACE>/g' operator.yaml
```

Comment:
```suggestion

       sed -i -e 's/PLACEHOLDER-NAMESPACE/<YOUR NAMESPACE>/g' operator.yaml
```

I think the trailing whitespace in the line before is making this line appear quoted:

<img width="999" alt="image" src="https://user-images.githubusercontent.com/6631887/79918345-10259480-83e1-11ea-98d0-aa656c617bed.png">

Reply:
this is still getting rendered as a quote when I click "view file"